### DISCUSSION TITLE
reducing spaces in menus

### DISCUSSION THREAD

**Discussion by benutzer74** [tsujan/Kvantum#657] [Q&A] [answered]
For my taste, there is way to much space in menus (e.g. context menus, right mouse, not toolbar menu) in the KvCurvesLight theme. I looked for everything with "spacing" in KvCurvesLight#.kvconfig and reduced it, but there´s still too much space in the menus. How can I now reduce space? Do I need to change the fonts, does the menu spaces depend on font size? Thanks!

**Comment by tsujan** [comment, marked as answer]
All parameters are is explained in the documentation.

The height of a menu is calculated based on several factors, among which are `menu_separator_height` under `[%General]` and `frame.top`, `frame.bottom`, `text.margin.top` and `text.margin.bottom` under `[MenuItem]`.

Many parameters aren't included in Kvantum Manager because changing them freely may ruin a theme but it's good to see their effects in the `kvconfig` file by trial and error.

EDIT: Use `kvantumpreview` in a terminal after each change to see its effect.

**Reply by benutzer74** [reply]
Thanks. With the many options it´s hard to keep overview ;-)

**Reply by tsujan** [reply]
Yes, there are many parameters. Sometimes, I should read the documentation that I've written myself to find them.
